gdal/src/main/scala/geotrellis/contrib/vlm/gdal/GDALDataset.scala

```diff
@@ -68,7 +68,7 @@ case class GDALDataset(token: Long) extends AnyVal {
     val extent = this.extent(dataset)
 
     if (GDALWarp.get_overview_widths_heights(token, dataset, numberOfAttempts, 1, widths, heights) <= 0)
-      throw new Exception("get_overview_widths_heights")
+      throw new MalformedDataException("Unable to read in overviews")
```

Comment:
It can not construct overivew `RasterExtents`. Mb it is better to throw an exception that would explain the issue cause in terms of the entire function definition?